srcpkgs/dwarffortress/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'dwarffortress'
+pkgname=dwarffortress
+version=0.44.12
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+depends="gtk SDL SDL_ttf SDL_image libGL glu"
+short_desc="Control a dwarven outpost or an adventurer in a randomly generated, persistent world"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="custom: Proprietary"
+homepage="http://www.bay12games.com/dwarves/"
+distfiles="http://www.bay12games.com/dwarves/df_44_12_linux.tar.bz2"
+checksum=5a2d15730d21b41a68da09ce3c3b70c3e2a6fd15b95c6cbfb68578ed2d2db979
+nopie="distfiles are precompiled as PIE"
+repository="nonfree"
+
+do_patch() {
+	vsed -i df_linux/df -e "s;./libs/Dwarf_Fortress;LD_PRELOAD=/usr/lib/libstdc++.so.6 ./libs/Dwarf_Fortress;"
+}
+
+do_install() {
+	vbin ${FILESDIR}/dwarffortress
+	vmkdir /usr/share/dwarffortress
+	vmkdir /usr/lib/dwarffortress
+	vcopy "df_linux/libs/*" /usr/lib/dwarffortress/
```

Comment:
```
	vmkdir /usr/lib/dwarffortress/libs
	vcopy "df_linux/libs/*" /usr/lib/dwarffortress/libs
```

Reply:
what's the purpose of this?

Reply:
df seems to search for librarys in a folder called "libs" in a folder called libs.
whether this change or the wrapper script needs to copy files from /usr/lib/dwarffortress to ${XDG_DATA_HOME:-$HOME/.local/share}/dwarffortress/libs.
maybe the second is better. idk

Reply:
nvm I see the problem, thanks

Reply:
changing to symlink the libs, makes more sense since these are binaries and don't have any need to be copied to each user/instance. Good catch